app/models/barcode/format_handlers.rb

```diff
@@ -316,6 +316,15 @@ class Randox < BaseRegExBarcode
     self.format = /\A(?<prefix>\d{2}[A-Z]{3}\d{2})-(?<number>\d{4})(?<suffix>[A-Z])\z/
   end
 
+  # XXX-AA-NNNNNNNN
+  # where:
+  #  X = letter character A-Z
+  #  A = alphanumeric character A-Z/0-9
+  #  N = number character 0-9
+  class RandoxV2 < BaseRegExBarcode
+    self.format = /\A(?<prefix>[A-Z]{3}-[A-Z0-9]{2})-(?<number>\d{8})\z/
+  end
```

Comment:
Hmm, just checked the barcode that kicked this all off (first message in the RT https://rt.sanger.ac.uk/Ticket/Display.html?id=714381), and it has six digits rather than 8. 

That's probably worth clarifying at this stage (in email thread "Re: Heron/Ealgle barcode and file name standards" maybe?)